regression/cuda/benchmarks/114_cudnn_conv/main.cu

```diff
@@ -0,0 +1,41 @@
+#include <cudnn.h>
+#include <assert.h>
+
+int main()
+{
+    cudnnHandle_t handle;
```

Comment:
Can you please add some comments?

What is the purpose of this regression test? What are you testing?

Reply:
I've added some notes in the comments.